ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productteaser/v1/productteaser/_cq_dialog/.content.xml

```diff
@@ -1,61 +1,70 @@
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                                fieldDescription="The call to action to include in the teaser, if any. Note the the 'Add to cart' option only works for simple/variant products"
+                                                fieldLabel="Call to action"
+                                                name="./cta">
                                                 <items jcr:primaryType="nt:unstructured">
-                                                    <none jcr:primaryType="nt:unstructured"
-                                                          selected="{Boolean}true"
-                                                          text="None"
-                                                          value=""/>
-                                                    <add-to-cart jcr:primaryType="nt:unstructured"
-                                                                 text="Add to cart"
-                                                                 value="add-to-cart"/>
-                                                    <details jcr:primaryType="nt:unstructured"
-                                                             text="See more details"
-                                                             value="details"/>
+                                                    <none
+                                                        jcr:primaryType="nt:unstructured"
+                                                        selected="{Boolean}true"
+                                                        text="None"
+                                                        value=""/>
+                                                    <add-to-cart
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="Add to cart"
+                                                        value="add-to-cart"/>
+                                                    <details
+                                                        jcr:primaryType="nt:unstructured"
+                                                        text="See more details"
+                                                        value="details"/>
                                                 </items>
                                             </call-to-action>
+                                            <ctaText
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                                                fieldDescription="The text displayed by the call-to-action button"
+                                                fieldLabel="Call to action: text"
```

Comment:
`action:` why is there a colon here in the label?

Reply:
Just because I added it for the label: I can remove it, no problem.